content/articles/2019-06-30-end-of-first-gsoc-phase.md

```diff
@@ -0,0 +1,57 @@
+Title: End of first GSoC phase 
+Date: 2019-06-30 11:30
+Category: GSCOC
+Tags: GSOC, GSOC19, poliastro
+slug: 2019-06-30-end-of-first-gsoc-phase
```

Comment:
My only comment is to find a more descriptive title, what about "converting-to-czml-and-next-steps"? (You would also have to change the `Title` and the filename